sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeSuite.scala

```diff
@@ -132,4 +135,33 @@ class ExchangeSuite extends SparkPlanTest with SharedSQLContext {
     val projection2 = cached.select("_1", "_3").queryExecution.executedPlan
     assert(!projection1.sameResult(projection2))
   }
+
+  test("SPARK-26601: Make broadcast-exchange thread pool configurable") {
+    val previousNumber = SparkSession.getActiveSession.get.sparkContext.conf
+      .get(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER)
+
+    SparkSession.getActiveSession.get.sparkContext.conf.
+      set(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER, 1)
+    assert(SQLConf.get.getConf(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER) === 1)
+
+    Future {
+      Thread.sleep(5*1000)
+    } (BroadcastExchangeExec.executionContext)
+
+    val f = Future {} (BroadcastExchangeExec.executionContext)
```

Comment:
You don't have to test Java's thread executors. Can you just check if `BroadcastExchangeExec.executionContext .getMaximumPoolSize` is as configured?

Reply:
Actually, i have thought about this method. But i can not call `getMaximumPoolSize` api, then i change the solution.
I will search more about this idea